internal/cmd/gocdk/_static/biome_add/outputs.tf

```
@@ -0,0 +1,10 @@
+# This file specifies Terraform module outputs.
+# See https://www.terraform.io/docs/configuration/outputs.html
+
+# The launch specifier sets options for the biome's launcher.
+# This is a local launcher, which runs on the local Docker daemon.
+output "launch_specifier" {
+	value {
+		host_port = 8080
```

As mentioned over IM, please set `project_id`, `location` (should always be `us-central1` in current Cloud Run beta), and `service_name`.

```suggestion
		project_id   = "" # fill this in
		location     = "us-central1"
		service_name = "" # fill this in
```